scripts/auto_schedule_fixes.py

```diff
@@ -33,7 +33,18 @@ def generate_speed_geojson(n, gdf, output_dir, filename_suffix):
 
 if __name__ == '__main__':
     arg_parser = argparse.ArgumentParser(
-        description=''
+        description='Script to check and correct, if needed, the speed and headway of services in the schedule.'
```

Comment:
I prefer descriptions to break over fewer lines (see changes in #188). This will be automaticall wrapped when calling `--help` in the CLI anyway. I would break only where natural. You can also use `""" ... """` to do it all as one big string block. It's then easier to read when maintaining.

Reply:
ok, I will keep that in mind when I port this to cli in your branch 👍